statsmodels/genmod/cov_struct.py

```diff
@@ -938,6 +951,14 @@ class Equivalence(CovStruct):
     using this covariance structure, missing data should be removed
     before constructing the model.
 
+    If using `labels`, after a model is defined using the covariance
+    structure it is possible to remove a label pair from the second
+    level of the `pairs` dictionary to force the corresponding
+    covariance to be zero.
```

Comment:
Thanks that's more understandable.
Is there an easier, more official way for users to do this, other than directly manipulating the pairs dictionary?
For example, is there a zero correlation label? `
or maybe we should have a `zero_label_pairs` keyword for those `b/c` combinations with zero correlation. (sounds more like an additional enhancement that might not be necessary for this PR.)

(BTW: when replying to github comments by email, please remove the quoted original text unless it is a used as a direct reference.)